prow/jobs/kyma/tools/load-test/load-test.yaml

```diff
@@ -52,6 +52,16 @@ presubmits: # runs on PRs
       labels:
         <<: *job_labels_template
         preset-build-pr: "true"
+    - name: pre-rel08-kyma-tools-load-test
```

Comment:
This doesn't have `always_run: true`. Could you make sure if this was intended or a mistake?

Reply:
this has been missed already in 07, i added it to the config